tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -529,6 +534,16 @@ public bool DiagnosticSourceEnabled
         /// </summary>
         internal bool IsRareSamplerEnabled { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the tracer is running in AAS
+        /// </summary>
+        internal bool InAzureAppService { get; set; }
```

Comment:
nit: I would rename it to start with `Is`. Something like `IsInAzureAppService` or `IsRunningInAzureAppService`

Reply:
Done in 11bd7370730cb3629c42f1dc0b13322e2610e631